_tasks/dnt.ts

```diff
@@ -136,8 +137,21 @@ await Promise.all([
   fs.copy("server/static/", path.join(outDir, "esm/server/static/")),
 ])
 
-await Deno.writeTextFile(
-  "target/npm/esm/main.js",
-  (await Deno.readTextFile("target/npm/esm/main.js"))
-    .replace(/^#!.+/, "#!/usr/bin/env -S node --loader ts-node/esm"),
-)
+await Promise.all([
+  editFile(
+    "target/npm/esm/main.js",
+    (content) =>
+      content
+        .replace(/^#!.+/, "#!/usr/bin/env -S node --loader ts-node/esm"),
```

Comment:
Should we add `ts-node` as a peer dependency?

Reply:
I'm not sure if there's a way to add a peer dev dependency, which is what we'd want here.

Reply:
Yes, but I believe that will force downstream users to add `ts-node` as a non-dev dependency, when it should be a dev dependency.